packages/core/src/components/slider/handle.tsx

```diff
@@ -67,28 +66,24 @@ export class Handle extends AbstractPureComponent2<IInternalHandleProps, IHandle
     }
 
     public render() {
-        const { className, disabled, label, min, tickSizeRatio, value, vertical } = this.props;
+        const { className, disabled, label } = this.props;
         const { isMoving } = this.state;
 
         // The handle midpoint of RangeSlider is actually shifted by a margin to
         // be on the edge of the visible handle element. Because the midpoint
         // calculation does not take this margin into account, we instead
         // measure the long side (which is equal to the short side plus the
         // margin).
```

Comment:
this comment needs to be moved to the new helper function `getStyleProperties()`

Reply:
✅